src/version.coffee

```diff
@@ -1 +1 @@
-module.exports = '0.7.8'
+module.exports = '0.7.11'
```

Comment:
You could do `require('../package.json').version` instead of manually bumping versions in several places if you like.